apps/financial-aid/web-osk/src/routes/application/TaxReturnForm/taxReturnForm.tsx

```diff
@@ -28,59 +57,70 @@ const TaxReturnForm = () => {
-        <Text as="h1" variant="h2" marginBottom={2}>
+        <Text as="h1" variant="h2" marginBottom={4}>
           Skattagögn
         </Text>
 
-        <Text marginBottom={2}>
-          Við þurfum að fá afrit af nýjasta <strong>skattframtali</strong> þínu
-          og staðfestingarskjal úr <strong>staðreiðsluskrá</strong> Skattsins.
-        </Text>
+        {taxReturnFetchFailed && (
+          <Box marginBottom={4}>
+            <AlertMessage
+              type="warning"
+              title="Ekki tókst að sækja skattframtal"
+              message="Þú getur hlaðið upp skattframtalinu þínu hérna ásamt staðgreiðsluskrá"
+            />
+          </Box>
+        )}
 
-        <Text marginBottom={[4, 4, 5]}>
-          Við þurfum að fá afrit af nýjasta skattframtali þínu. Skattframtal er
-          staðfesting á öllum þeim tekjum, eignum og skuldum sem þú áttir á
-          skattárinu sem leið og er nauðsynlegt fylgigagn fyrir úrvinnslu á
-          fjárhagsaðstoð.
-        </Text>
+        {content.data}
+
+        {content.reason}
 
         <Files
           header="Dragðu gögn hingað"
           fileKey="taxReturnFiles"
           uploadFiles={form.taxReturnFiles}
         />
 
-        <Text as="h2" variant="h3" marginBottom={2}>
-          Hvar finn ég staðfest afrit af mínu skattframtali?
-        </Text>
-
-        <LinkContext.Provider
-          value={{
-            linkRenderer: (href, children) => (
-              <a
-                style={{
-                  color: '#0061ff',
-                }}
-                href={href}
-                rel="noopener noreferrer"
-                target="_blank"
-              >
-                {children}
-              </a>
-            ),
-          }}
-        >
-          <Text marginBottom={[3, 3, 5]}>
-            Á vef Skattsins finnur þú{' '}
-            <a href="https://www.skatturinn.is/einstaklingar/framtal-og-alagning/stadfest-afrit-framtals/">
-              leiðbeiningar
-            </a>{' '}
-            um hvernig sækja má staðfest afrit skattframtals.
-          </Text>
-        </LinkContext.Provider>
+        {taxReturnFetchFailed && (
+          <>
+            <Text as="h2" variant="h3" marginBottom={2}>
+              Hvar finn ég staðfest afrit af mínu skattframtali?
+            </Text>
+
+            <LinkContext.Provider
+              value={{
+                linkRenderer: (href, children) => (
+                  <a
+                    style={{
+                      color: '#0061ff',
+                    }}
+                    href={href}
+                    rel="noopener noreferrer"
+                    target="_blank"
+                  >
+                    {children}
+                  </a>
+                ),
+              }}
+            >
+              <Text marginBottom={[3, 3, 5]}>
+                Á vef Skattsins finnur þú{' '}
+                <a href="https://www.skatturinn.is/einstaklingar/framtal-og-alagning/stadfest-afrit-framtals/">
```

Comment:
if I press this will it lead me there or open in another tab?

Reply:
yes it does